docs/models/data-types.mdx

```diff
@@ -185,7 +186,7 @@ Numeric options can be combined:<br/>
 

```

Comment:
For line 160, it needs to point towards `#approximate-decimal-numbers` instead of `inexact`